### ISSUE TITLE
loosen model of nomenclature-citation

### ISSUE THREAD

**Issue by tcatapano** [plazi/TaxPub#34]
current model:

```
(tp:taxon-name, tp:taxon-author*, (tp:type-genus | tp:type-species)?, 
(mixed-citation | xref)?, tp:material-citation*, tp:taxon-status?, comment?) 

```

**Comment by tcatapano** [comment]
see https://github.com/plazi/TaxPub/blob/794f337abd45d31d88cfab84680f37177c32b879/taxpubcustom-elements-NS0-v1.ent#L122

**Comment by myrmoteras** [comment]
@myrmoteras to add some samples to test this change.
EJT example
Pensoft taxpub

**Comment by tcatapano** [comment]
strange example in  http://treatment.plazi.org/GgServer/xml/03A10B47FFEBFF93FDB2FBB6FB4DFBF5

treatmentCitationGroup
-- taxonomicName
--- treatmentCitation
-- text

TreatmentCitation inside taxonomicName

```
<subSubSection box="[189,1331,1184,1212]" pageId="19" pageNumber="20" type="reference_group">
--
  | <paragraph blockId="19.[189,1331,1184,1212]" box="[189,1331,1184,1212]" pageId="19" pageNumber="20">
  | <treatmentCitationGroup box="[189,1331,1184,1212]" pageId="19" pageNumber="20">
  | <taxonomicName authority="Kraus, 1958: 3" authorityName="Kraus" authorityPageNumber="3" authorityYear="1958" box="[189,661,1185,1212]" class="Diplopoda" family="Paradoxosomatidae" genus="Eviulisoma" higherTaxonomySource="GBIF" kingdom="Animalia" order="Polydesmida" pageId="19" pageNumber="20" phylum="Arthropoda" rank="species" species="kwabuniense">
  | <treatmentCitation author="" box="[189,661,1185,1212]" page="3" pageId="19" pageNumber="20">
  | <emphasis box="[189,480,1185,1211]" italics="true" pageId="19" pageNumber="20">Eviulisoma kwabuniense</emphasis>
  | <bibRefCitation box="[487,635,1185,1212]" pageId="19" pageNumber="20" refId="ref33917" refString="Kraus O. 1958. Myriapoden aus Ostafrika (Tanganjika-Territory). Veroffentlichungen aus dem Uberseemuseum Bremen (A) 3: 1 - 16." type="journal article">Kraus, 1958</bibRefCitation>
  | : 3
  | </treatmentCitation>
  | </taxonomicName>
  | (holotype (not studied) in the Überseemuseum Bremen).
  | </treatmentCitationGroup>
  | </paragraph>
  | </subSubSection>


```

**Comment by tcatapano** [comment]
more regular example:

treatmentCitationGroup
- taxonomicName
- treatmentCitation
--- bibrefCitation
- text

```
subSubSection box="[189,1319,1642,1670]" pageId="46" pageNumber="47" type="reference_group">
<paragraph blockId="46.[189,1319,1642,1670]" box="[189,1319,1642,1670]" pageId="46" pageNumber="47">
<treatmentCitationGroup box="[189,1319,1642,1670]" pageId="46" pageNumber="47">
<taxonomicName authority="Kraus, 1958: 2" authorityName="Kraus" authorityPageNumber="2" authorityYear="1958" box="[189,650,1643,1670]" class="Diplopoda" family="Paradoxosomatidae" genus="Eviulisoma" higherTaxonomySource="GBIF" kingdom="Animalia" order="Polydesmida" pageId="46" pageNumber="47" phylum="Arthropoda" rank="species" species="dabagaense">
<emphasis box="[189,468,1643,1669]" italics="true" pageId="46" pageNumber="47">Eviulisoma dabagaense</emphasis>
<treatmentCitation author="" box="[476,650,1643,1670]" page="2" pageId="46" pageNumber="47">
<bibRefCitation box="[476,623,1643,1670]" pageId="46" pageNumber="47" refId="ref33917" refString="Kraus O. 1958. Myriapoden aus Ostafrika (Tanganjika-Territory). Veroffentlichungen aus dem Uberseemuseum Bremen (A) 3: 1 - 16." type="journal article">Kraus, 1958</bibRefCitation>
: 2
</treatmentCitation>
</taxonomicName>
(holotype (not studied) in the Überseemuseum Bremen).
</treatmentCitationGroup>
</paragraph>
</subSubSection>
```

**Comment by tcatapano** [comment]
see: https://github.com/plazi/stable-treatment-html/blob/9de18677037e50fbb21b565dad57c89e23f1d4a7/data/038C7E3DFF86C42760ACFD701E1CFC73.xml#L88


bibRefCitations inside of bibRefCitations...
```
<treatmentCitation author="" box="[407,783,827,854]" page="56" pageId="15" pageNumber="16">
<bibRefCitation box="[407,740,827,853]" pageId="15" pageNumber="16" refId="ref13080" refString="Golovatch S. I. &amp; Enghoff H. 1994. Review of the dragon millipedes, genus Desmoxytes Chamberlin, 1923 (Diplopoda, Polydesmida, Paradoxosomatidae). Steenstrupia 20 (2): 45 - 71." type="journal article">
Golovatch &amp; 
<bibRefCitation box="[567,740,827,853]" pageId="15" pageNumber="16" refId="ref13080" refString="Golovatch S. I. &amp; Enghoff H. 1994. Review of the dragon millipedes, genus Desmoxytes Chamberlin, 1923 (Diplopoda, Polydesmida, Paradoxosomatidae). Steenstrupia 20 (2): 45 - 71." type="journal article">
Enghoff, 
<bibRefCitation box="[677,740,827,853]" pageId="15" pageNumber="16" refId="ref13080" refString="Golovatch S. I. &amp; Enghoff H. 1994. Review of the dragon millipedes, genus Desmoxytes Chamberlin, 1923 (Diplopoda, Polydesmida, Paradoxosomatidae). Steenstrupia 20 (2): 45 - 71." type="journal article">1994</bibRefCitation>
</bibRefCitation>
</bibRefCitation>
: 56
</treatmentCitation>
```

**Comment by tcatapano** [comment]
another example with material info in treatmentCitationGroup:

https://github.com/plazi/stable-treatment-html/blob/master/data/215187C84C11B318FF6BFA3DF60BD400.xml

```
<treatmentCitationGroup pageId="61" pageNumber="62">
<materialsCitation box="[151,1096,799,824]" country="Panama" elevation="229" location="Bugaba" pageId="61" pageNumber="62" specimenCount="1" typeStatus="holotype">
<taxonomicName authority="Sharp 1885: 408" authorityName="Sharp" authorityPageNumber="408" authorityYear="1885" box="[151,582,799,824]" class="Insecta" family="Staphylinidae" genus="Philonthus" higherTaxonomySource="GBIF" kingdom="Animalia" order="Coleoptera" pageId="61" pageNumber="62" phylum="Arthropoda" rank="species" species="cavifrons">
<emphasis box="[151,383,799,824]" italics="true" pageId="61" pageNumber="62">Philonthus cavifrons</emphasis>
<treatmentCitation author="" box="[392,582,799,824]" page="408" pageId="61" pageNumber="62">
<bibRefCitation box="[392,525,799,824]" pageId="61" pageNumber="62" refId="ref72492" refString="Sharp, D. (1885) Staphylinidae. Biologia Centrali-Americana. Insecta. Coleoptera, 1 (2), 393 - 536. [http: // www. sil. si. edu / digitalcollections / bca /]" type="journal article">Sharp 1885</bibRefCitation>
: 408
</treatmentCitation>
</taxonomicName>
. 
<typeStatus box="[595,650,800,824]" pageId="61" pageNumber="62">Type</typeStatus>
locality: 
<collectingCountry box="[758,850,800,823]" name="Panama" pageId="61" pageNumber="62">Panama</collectingCountry>
, 
<location LSID="urn:lsid:plazi:treatment:215187C84C11B318FF6BFA3DF60BD400:AC2760054C15B375FCA2FCCBF4E6D65C" box="[862,947,799,824]" country="Panama" name="Bugaba" pageId="61" pageNumber="62">Bugaba</location>
80 
<elevation box="[984,1096,799,824]" metricMagnitude="2" metricUnit="m" metricValue="2.286" metricValueMax="4.572" metricValueMin="0.0" pageId="61" pageNumber="62" unit="ft" value="750.0" valueMax="1500.0" valueMin="0.0">
<quantity box="[984,1096,799,824]" metricMagnitude="2" metricUnit="m" metricValue="2.286" metricValueMax="4.572" metricValueMin="0.0" pageId="61" pageNumber="62" unit="ft" value="750.0" valueMax="1500.0" valueMin="0.0">0–1500 ft</quantity>
</elevation>
</materialsCitation>
. 
<materialsCitation collectionCode="BMNH" country="Panama" location="Distribution" pageId="61" pageNumber="62" specimenCount="1" specimenCount-male="1" typeStatus="holotype">
<typeStatus box="[1108,1163,800,824]" pageId="61" pageNumber="62">Type</typeStatus>
depository: 
<collectionCode box="[1306,1390,800,823]" pageId="61" pageNumber="62">BMNH</collectionCode>
HT 
<specimenCount box="[197,221,837,860]" pageId="61" pageNumber="62" type="male">♂</specimenCount>
. 
<location LSID="urn:lsid:plazi:treatment:215187C84C11B318FF6BFA3DF60BD400:AC2760054C15B375FF14FC90F621D639" box="[232,372,836,861]" country="Panama" name="Distribution" pageId="61" pageNumber="62">Distribution</location>
: 
<collectingCountry box="[382,474,837,860]" name="Panama" pageId="61" pageNumber="62">Panama</collectingCountry>
</materialsCitation>
.
</treatmentCitationGroup>
```

**Comment by myrmoteras** [comment]
this could or should be used as an example: 
http://treatment.plazi.org/id/C3635F69FFC6980DFD93F915FE06FBAE

https://github.com/plazi/EJT-testbed/blob/master/incoming/ejt-502_trietsch_miko_deans_xml_id.xml

**Comment by tcatapano** [comment]
Now much looser with: ab7d3d6 

https://github.com/plazi/TaxPub/blob/ab7d3d66165354f240b48b3ca5fe50820f2c7655/taxpubcustom-elements-NS0-v1.ent#L122

basically a mixed-nomenclature-citation allowing anything from text only to element-only. Could be too loose as it allows multiple taxon-names etc which may not make sense in an authoring context.

**Comment by tcatapano** [comment]
Sufficiently implemented for 1.0 milestone. Closing.
